.github/workflows/tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   build_ubuntu:
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

Comment:
Was the `-dev` suffix still necessary, or is `3.12` enough nowadays?

(Can be addressed in a separate PR if needed.)

Reply:
Yeah, let's remove it.